### DISCUSSION TITLE
Can I simply called chart or stats from custom page like table and form?

### DISCUSSION THREAD

**Discussion by valpuia** [filamentphp/filament#6124] [Help]
In custom page we can implement tables and form and can easily called using `{{ $this->table }}` or `{{ $this->form }}` from blade file, but for chart and stats I have to called like below from PageResource:

```
protected function getHeaderWidgets(): array
{
    return [
        ExampleStatsOverview::class
    ];
}

protected function getFooterWidgets(): array
{
    return [
        ExampleBarChart::class
    ];
}
```

If i want to display `table` and `stats/chart` in grid like below, how to implement?

```
<div class="grid grid-cols-3 gap-4">
    <div class="col-span-2">{{ $this->table }}</div>
    <div>I want to display stats here</div>
</div>
```
Is there any logic I can call chart or stats data from blade file like {{ $this->chart }}, {{ $this->widget }} ?

**Comment by danharrin** [comment]
Widgets are Livewire components, so you could render them with `@livewire(\Fully\Qualified\Widget\Class\Name\Here::class)`

**Reply by valpuia** [reply]
I called like this `@livewire(\App\Filament\Resources\PostManagement\PostResource\Widgets\PostDetailOverview::class)` and it throws error:

`Unable to find component: [app.filament.resources.post-management.post-resource.widgets.post-detail-overview]`

And my stats doesnot extend component

```
class PostDetailOverview extends BaseWidget
{
......
}
```

If I called like this from Custom page then it is working.. but cannot directly from custom.blade.php

```
protected function getHeaderWidgets(): array
{
    return [
        \App\Filament\Resources\PostManagement\PostResource\Widgets\PostDetailOverview::class
    ];
}
```

**Reply by danharrin** [reply]
I don't know then... Livewire should register that component. Maybe try registering it manually in a service provider?

**Comment by FrazeColder** [comment]
Any news here? I have the same problem

**Reply by danharrin** [reply]
Where is your widget located
